pkg/proxy/nftables/proxier.go

```diff
@@ -1015,6 +1038,94 @@ func isAffinitySetName(set string) bool {
 	return strings.HasPrefix(set, servicePortEndpointAffinityNamePrefix)
 }
 
+// nftElementStorage is an internal representation of nftables map with single string value or set.
+type nftElementStorage struct {
+	elements      map[string]string
+	leftoverKeys  sets.Set[string]
+	containerType string
+	containerName string
+}
+
+func (c *nftElementStorage) joinKey(k []string) string {
+	return strings.Join(k, " . ")
+}
+
+func (c *nftElementStorage) splitKey(k string) []string {
+	return strings.Split(k, " . ")
```

Comment:
these are helper functions, no need to be part of the type

Reply:
yeah I only did that to give it a context of `nftElementStorage`, considering no one else should really need these functions. Do you prefer them being separate?

Reply:
Given that they neither read from nor modify the `nftElementStorage` itself, I agree they would make more sense as just helper functions.

(Also, maybe add a comment that this is converting them into the representation that will be used when they're passed to `nft`, so we know that the parsing is going to be unambiguous.)